tests/stubtest_stdlib.py

```diff
@@ -27,6 +27,7 @@ def run_stubtest(typeshed_dir: Path) -> int:
         "-m",
         "mypy.stubtest",
         "--check-typeshed",
+        "--show-traceback",
```

Comment:
I assume this is a testing remnant?

Reply:
It was for testing but it seemed harmless to keep in.